contracts/netmap/contract.go

```diff
@@ -421,7 +428,8 @@ func NewEpoch(epochNum int) {
 
 	// todo: check if provided epoch number is bigger than current
 	storage.Put(ctx, snapshotEpoch, epochNum)
-	storage.Put(ctx, snapshotBlockKey, ledger.CurrentIndex())
+	curIndex := ledger.CurrentIndex()
+	storage.Put(ctx, snapshotBlockKey, curIndex)
```

Comment:
Do we need this key then? It can be derived from the new index.

Reply:
technically we dont, it's accessible via 2 `storage.Get` ops. Worth dropping?

Reply:
It's not often used and for an RPC invocation the difference in access time is nothing. But it's one key less in the DB, always good.

Reply:
done